recipes/libnsl/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "libnsl" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/thkukuk/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.xz
+  sha256: 2da075ef1893ebdfc5f074f83ac811873dc06fd5c62bc9a4729fd2e27a40341a
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage('libnsl', max_pin='x.x') }}
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - pkg-config
+    - libtool
+    - autoconf
+  host:
+    - glib
+    - libtirpc
+  run:
+    - glib
+    - libtirpc
```

Comment:
If you leave these as dependencies, we are not going to have this as a dependency of python etc as this would bring a lot of dependencies to python.
libnsl I see on ubuntu doesn't have these dependencies.
```
$ ldd /usr/lib/x86_64-linux-gnu/libnsl.so
        linux-vdso.so.1 (0x00007ffe2018a000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f414817c000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f4148787000
```

Reply:
I was following the Arch linux recipe, but it looks like libtirpc is optional. I also mistook glibc for glib, so that is not needed either. I removed both and it looks like it builds OK. Thanks for pointing this out!